src/components/ui/TripCard/TripCard.styled.ts

```diff
@@ -0,0 +1,40 @@
+import styled from '@emotion/styled';
+
+export const TripCardStyled = styled('section')(({ theme }) => ({
+    display: 'flex',
+    flexDirection: 'column',
+    gap: '2rem',
+    backgroundColor: `${theme.color.background}`, // Placeholder for dynamic theme value
```

Comment:
Here Template Literal (`backtick`) is not required since  it's key:value pair. So you can update it like

`backgroundColor: theme.color.background`